src/simplicial-hott/08-covariant.rzk.md

```diff
@@ -14,6 +14,7 @@ This is a literate `rzk` file:
   instance the notion of contractible types.
 - `3-simplicial-type-theory.md` — We rely on definitions of simplicies and their
   subshapes.
+- `4-extension-types.md` — We use Theorem 4.1, an equivalence between lifts.  
```

Comment:
I assume, you would also want `WeakExt` from there after #42 is merged.

Reply:
I agree the WeakExt I wrote was temporary and meant to replaced by the appropriate one in the right place. I just didn't where that would be. So, yeah we should certainly adjust it.